src/main/java/net/minecraftforge/fmp/ForgeMultipartModContainer.java

```diff
@@ -0,0 +1,162 @@
+import java.util.Map;
+
+import com.google.common.eventbus.EventBus;
+import com.google.common.eventbus.Subscribe;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.common.ForgeVersion;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.client.registry.ClientRegistry;
+import net.minecraftforge.fml.common.DummyModContainer;
+import net.minecraftforge.fml.common.FMLCommonHandler;
+import net.minecraftforge.fml.common.LoadController;
+import net.minecraftforge.fml.common.ModMetadata;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLMissingMappingsEvent;
+import net.minecraftforge.fml.common.event.FMLMissingMappingsEvent.MissingMapping;
+import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.network.NetworkCheckHandler;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fml.common.registry.GameRegistry.Type;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+import net.minecraftforge.fmp.block.BlockMultipartContainer;
+import net.minecraftforge.fmp.block.TileCoverable;
+import net.minecraftforge.fmp.block.TileMultipartContainer;
+import net.minecraftforge.fmp.capabilities.CapabilityItemSaw;
+import net.minecraftforge.fmp.capabilities.CapabilityMicroblockContainerTile;
+import net.minecraftforge.fmp.capabilities.CapabilityMultipartContainer;
+import net.minecraftforge.fmp.capabilities.CapabilitySlottedCapProvider;
+import net.minecraftforge.fmp.client.multipart.MultipartContainerSpecialRenderer.TileCoverableSpecialRenderer;
+import net.minecraftforge.fmp.client.multipart.MultipartContainerSpecialRenderer.TileMultipartSpecialRenderer;
+import net.minecraftforge.fmp.client.multipart.MultipartStateMapper;
+import net.minecraftforge.fmp.event.FMPEventHandler;
+import net.minecraftforge.fmp.network.MultipartNetworkHandler;
+
+public final class ForgeMultipartModContainer extends DummyModContainer
+{
+    public static final String MODID = "forgemultipart";
+
+    public static BlockMultipartContainer multipart;
+    public static boolean registered = false;
+
+    public ForgeMultipartModContainer()
+    {
+        super(new ModMetadata());
+        ModMetadata meta = getMetadata();
+        meta.modId = MODID;
+        meta.name = "Forge Multipart";
+        meta.version = ForgeVersion.getVersion();
+        meta.credits = "Made possible with help from everybody in #MCMultiPart";
+        meta.authorList = Arrays.asList("amadornes");
+        meta.description = "Forge Multipart provides the ability for mods to have more than one \"block\" " +
+                "in the same blockspace. ";
+        meta.url = "https://github.com/MinecraftForge/comingsoon/wiki";
```

Comment:
The API does not need it's own mod container.
The FML one is in for purely legacy reason and will *eventually* be going away.